Boa.Constrictor/WebDriver/Tasks/Check.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Boa.Constrictor.Screenplay;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Interactions;
+
+namespace Boa.Constrictor.WebDriver
+{
+    /// <summary>
+    /// Checks an element if not already selected
+    /// Useful for checkboxes or radio buttons
+    /// </summary>
+    public class Check : AbstractWebLocatorTask
```

Comment:
I'm conflicted on the names for the `Check` and `Uncheck` Tasks:

- `Check.On(locator)`
- `Uncheck.On(locator)`

These two Tasks are nearly identical other then `true`/`false` in the `PerformAs` methods. To avoid duplication, we could combine them into one Task with a property that stores the expectation as a Boolean property. Again, naming is difficult. We could do something like `Check.On(locator)` / `Check.Off(locator)` or `Checkbox.Check(locator)` / `Checkbox.Uncheck(locator)`.

I'm not sure how to proceed. Thoughts, @thePantz ?

Reply:
yes, I agree we could reduce the duplication. 

Out of those options I prefer `Check.On()` / `Check.Off` since this task could be applied to more than just checkboxes.
I was thinking `Toggle.On` / `Toggle.Off` might be a more appropriate name

Reply:
I vote for `Check.On(locator)` and `Check.Off(locator)`.